prow/monitoring/README.md

```diff
@@ -0,0 +1,30 @@
+# Monitoring
+
+This folder contains the manifest files for monitoring prow resources.
+
+## Deploy
+
+TODO: The deployment has been integrated into our CI system. 
+
+A successful deploy will spawn a stack of monitoring for prow in namespace `prow-monitoring`: _prometheus_, _alertmanager_, and _grafana_.
+
+## Add more dashboards
```

Comment:
The sub-headings (`###`) are useful but this one I am not sure adds any value

Reply:
Removed.